dask/dataframe/tests/test_dataframe.py

```diff
@@ -4254,6 +4254,21 @@ def test_idxmaxmin_empty_partitions():
         ddf.b.idxmax().compute()
 
 
+def test_mode_numeric_only():
+    df = pd.DataFrame(
+        {
+            "int": [1, 2, 3, 4, 5, 6, 7, 8],
+            "float": [1.0, 2.0, 3.0, 4.0, np.nan, 6.0, 7.0, 8.0],
+            "dt": [pd.NaT] + [datetime(2010, i, 1) for i in range(1, 8)],
+            "timedelta": pd.to_timedelta([1, 2, 3, 4, 5, 6, 7, np.nan]),
+        }
+    )
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    assert_eq(ddf.mode(numeric_only=False), df.mode(numeric_only=False))
```

Comment:
Might be good to test that we have the same default (I've not actually run the suggestion below -- am just assuming it passes)

```suggestion
    assert_eq(ddf.mode(), df.mode())
    assert_eq(ddf.mode(numeric_only=False), df.mode(numeric_only=False))
```

Reply:
Passes locally